src/app/components/dropdown/index.html

```diff
@@ -1,15 +1,176 @@
+      </sky-docs-visual-example>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>For a dropdown button with one type of action, use the following format:</p>
+          <p>Button text: <span class="sky-emphasized">&lt;Verb&gt;</span></p>
+          <p>Menu item: <span class="sky-emphasized">&lt;Direct object&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/dropdown-single-action.png" alt="dropdown menu Edit: Addresses, Email addresses, Phone numbers" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>For a dropdown button with more than one type of action, use the following format:</p>
+          <p>Button text: <span class="sky-emphasized">More</span></p>
+          <p>Button icon: <span class="sky-emphasized">fa-ellipsis-h</span></p>
+          <p>Menu item: <span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/more-button.png" alt="dropdown menu More: Place vendor on hold, Archive vendor" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      
+        <div class="sky-text-block sky-margin-stacked-content-section">
+        <p>Additional button text guidelines:</p>
+				<ul>
+					<li>Use sentence-style capitalization.</li>
+					<li>The direct object can be singular or plural depending on whether the action acts on one or multiple items.</li>
+					<li>Don’t append extra words such as “details.” For example, use “Edit constituent” rather than “Edit constituent details.”</li>
+				</ul>
+      </div>
+    </sky-demo-page-content>
+      
+    <sky-demo-page-content sectionHeading="Layout">
+        
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Place context menus to the left of all row-specific content, but to the right of all list controls (such as checkboxes in a grid or repeater, or the expand-collapse actions in a tree view).</p>
+            <p>Default dropdowns are positioned like SKY UX buttons, with the same alignment and margins.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/dropdown/layout.png" />
+            <div class="caption caption-success sky-text-success">Do put context menu dropdowns to the left of all row-specific content in lists.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Related information">
+		<h3>Components</h3>
+			<ul>
+				<li><a routerLink="/components/button">Buttons</a></li>
```

Comment:
We mention popovers higher up, so I'm pretty sure they should be in this list. I'll add an entry.

Reply:
That's definitely a good call. I think we just missed that.

Reply:
Resolving...